resolwe_bio/docker_images/rnaseq/packages-manual/trimmomatic.sh

```diff
@@ -10,12 +10,19 @@ download_and_verify \
     http://www.usadellab.org/cms/uploads/supplementary/Trimmomatic/Trimmomatic-\${version}.zip \
     Trimmomatic-\${version}
 
-cat >trimmomatic.sh <<EOL
+cat <<EOF >trimmomaticSE
```

Comment:
To keep consistent we should name it `TrimmomaticSE` and `TrimmomaticPE`. As far as I tried it, it is still not working with input parameters?